.github/workflows/opened-pr-triage.yml

```
@@ -0,0 +1,14 @@
+name: Mark new pull request as waiting review
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  needs-triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: awaiting-review
```

```suggestion
          labels: waiting-review
```

I think this one is wrong